packages/mobile/src/types/analytics.ts

```diff
@@ -7,7 +7,11 @@ enum MobileEventNames {
   SHARE_TO_IG_STORY = 'Share to Instagram story - start',
   SHARE_TO_IG_STORY_CANCELLED = 'Share to Instagram story - cancelled',
   SHARE_TO_IG_STORY_ERROR = 'Share to Instagram story - error',
-  SHARE_TO_IG_STORY_SUCCESS = 'Share to Instagram story - success'
```

Comment:
ah interesting. I just added some events to track rate CTA stuff in common. Is there a difference adding the events here?

Reply:
Oh yeah these are mobile specific events which would have made sense for yours too @Kyle-Shanks

Reply:
gotcha. is it worth moving the review events here?

Reply:
Nah probably not